driver-core/src/test/unit/com/mongodb/client/model/changestream/ChangeStreamDocumentSpecification.groovy

```diff
@@ -35,8 +60,8 @@ class ChangeStreamDocumentSpecification extends Specification {
         def updateDesc = new UpdateDescription(['a', 'b'], BsonDocument.parse('{c: 1}'))
 
         when:
-        def changeStreamDocument = new ChangeStreamDocument<BsonDocument>(resumeToken, namespace, fullDocument, documentKey, clusterTime,
-                operationType, updateDesc)
+        def changeStreamDocument = new ChangeStreamDocument<BsonDocument>(resumeToken, namespaceDocument, fullDocument,
+                documentKey, clusterTime, operationType, updateDesc)
 
         then:
```

Comment:
Test the new properties: databaseName and namespaceDocument.

Reply:
Done